README.md

```diff
@@ -1,13 +1,14 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_tree-from-object-DOM/)
+   - [DEMO LINK](https://bogdana18.github.io/js_tree-from-object-DOM/)
```

Comment:
https://user-images.githubusercontent.com/38065883/174625410-49487f8c-af77-4771-91bc-dc98d6aac7b8.mov

DOM structure should look like this